elixir/exercicios/accumulate/.gitignore

```diff
@@ -0,0 +1,24 @@
+# The directory Mix will write compiled artifacts to.
```

Comment:
We could move the `.gitignore` one or two folders up. In that way we can reuse the same for all elixir related folders and keep everything more clean/tidy.